addons/mass_mailing/migrations/13.0.2.0/post-migration.py

```diff
@@ -8,7 +8,39 @@
 ]
 
 
+def fill_several_campaign_id(env):
+    openupgrade.logged_query(
+        env.cr, """
+    UPDATE mailing_mailing mm
```

Comment:
It seems the same query is repeated 4 times changing the table. This worths a loop iterating on the table name for a clearer code.

Reply:
done